### ISSUE TITLE
Restore error.

### ISSUE THREAD

**Issue by JerryLauzzz** [Russell91/TensorBox#280]
> * Tensorflow version : r1.3

https://github.com/Russell91/TensorBox/blob/master/train.py#L480

This do not work. 
When using pre-trained mobile-net,
```
        else :
            gvars = [x for x in tf.global_variables() if x.name.startswith(H['slim_basename']) and H['solver']['opt'] not in x.name]
            gvars = [x for x in gvars if not x.name.startswith("{}/AuxLogits".format(H['slim_basename']))]
            init_fn = slim.assign_from_checkpoint_fn(
                  '%s/data/%s' % (os.path.dirname(os.path.realpath(__file__)), H['slim_ckpt']),
                  gvars,
                  ignore_missing_vars=False)
            #init_fn = slim.assign_from_checkpoint_fn(
                  #'%s/data/inception_v1.ckpt' % os.path.dirname(os.path.realpath(__file__)),
                  #[x for x in tf.global_variables() if x.name.startswith('InceptionV1') and not H['solver']['opt'] in x.name])
            init_fn(sess)
```
This works.
Besides, in lstm_mobilenet_rezoom.json, image width, height should be specified.

**Comment by willSapgreen** [comment]
Hello @JerryLauzzz,

do you use mobilenet_v1_1.0_224 and what is your TensorFlow version?

I use mobilenet_v1_1.0_224 with TensorFlow 1.4.1,
but have this error
`Could not open MY_PATH/mobilenet_v1_1.0_224.ckpt.meta: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?`

Thank you for precious time to my question.

**Comment by willSapgreen** [comment]
@JerryLauzzz 's method works for me.
Also in lstm_mobilenet_rezoom.json,
"slim_ckpt": "mobilenet_v1_1.0_224.ckpt.meta" causes

> mobilenet_v1_1.0_224.ckpt.meta: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

on my side.
And it will be resolved by using "slim_ckpt": "mobilenet_v1_1.0_224.ckpt".

My TensorFlow version is 1.4.1.
